python/pyiceberg/cli/output.py

```diff
@@ -68,12 +68,18 @@ def uuid(self, uuid: Optional[UUID]):
 class ConsoleOutput(Output):
     """Writes to the console"""
 
+    def __init__(self, **properties: Any):
+        self.verbose = properties.get("verbose", False)
```

Comment:
When we use class variables, it is best to also add it to the class itself:
```python
class ConsoleOutput(Output):
    """Writes to the console"""

    verbose: bool
```

Reply:
Fixed.